packages/app/src/overview/components/backup-set-secret-key-modal-dialog/backup-set-secret-key-modal-dialog.component.tsx

```diff
@@ -43,6 +44,12 @@ const messages = defineMessages({
   backupSetSecretKeyModalInputLabel: {
     id: "module.overview.backupSetSecretKeyModalInputLabel",
   },
+  backupSetConfirmationSecretKeyModalInputLabel: {
+    id: "module.overview.backupSetConfirmationSecretKeyModalInputLabel",
+  },
+  backupPasswordConfirmationDoesntMatch: {
```

Comment:
🟡

```suggestion
  backupSecretKeyConfirmationDoesntMatch: {
```

`SecretKey` or `Password`. The password is a text so I understand the intention. However, I think that we should single semantic in a base code

Reply:
https://github.com/mudita/mudita-center/pull/518/commits/7a1d7d36113bcc0010d3fa2fd8708f496309465e